src/ru/desu/src/eu/kanade/tachiyomi/extension/ru/desu/Desu.kt

```diff
@@ -119,45 +98,37 @@ class Desu : ConfigurableSource, HttpSource() {
 
         var altName = ""
 
-        if (obj["synonyms"]?.jsonPrimitive?.content.orEmpty().isNotEmpty() && obj["synonyms"]!!.jsonPrimitive.contentOrNull != null) {
+        if (!synonyms.isNullOrEmpty()) {
             altName = "Альтернативные названия:\n" +
-                obj["synonyms"]!!.jsonPrimitive.content
-                    .replace("/", " / ") +
+                synonyms.replace("/", " / ") +
                 "\n\n"
         }
 
-        description = if (isEng.equals("rus")) {
-            obj["name"]!!.jsonPrimitive.content
-                .split(" / ")
-                .first()
-        } else {
-            obj["russian"]!!.jsonPrimitive.content
-                .split(" / ")
-                .first()
-        } + "\n" +
-            ratingStar + " " + ratingValue +
-            " (голосов: " +
-            obj["score_users"]!!.jsonPrimitive.int +
-            ")\n" + altName +
-            obj["description"]!!.jsonPrimitive.content
-
-        genre = if (chapter) {
-            "$category, $rawAgeStop, " +
-                obj["genres"]!!.jsonArray
-                    .map { it.jsonObject["russian"]!!.jsonPrimitive.content }
-                    .joinToString()
-        } else {
-            category + ", " + rawAgeStop + ", " + obj["genres"]!!.jsonPrimitive.content
-        }
-
-        status = when (obj["trans_status"]!!.jsonPrimitive.content) {
-            "continued" -> SManga.ONGOING
-            "completed" -> SManga.COMPLETED
-            else -> when (obj["status"]!!.jsonPrimitive.content) {
-                "ongoing" -> SManga.ONGOING
-                "released" -> SManga.COMPLETED
-                //  "copyright" -> SManga.LICENSED  Hides available chapters!
-                else -> SManga.UNKNOWN
+        val o = this
```

Comment:
you can just omit `this` when accessing members, like `image.original` instead of `this.image.original`;

so no need for this